crates/tessera-backbone/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tessera-abe = { workspace = true }
 rand = { workspace = true }
 base64 = { workspace = true }
 rmp-serde = { workspace = true }
+tessera-policy.path = "../tessera-policy"
```

Comment:
It would be more appropriate to include this in the workspace dependency. It would provides better visibility across the monorepo.